app/presenters/organisations/show_presenter.rb

```diff
@@ -119,8 +127,20 @@ def foi_title(title)
       I18n.t('organisations.foi.freedom_of_information_requests')
     end
 
-    def foi_description(contact)
-      content_tag(:p, contact.gsub("\r\n", "<br/>").html_safe) if contact
+    def foi_address(post)
+      data = ""
+      data << contact_line(post["title"])
+      data << contact_line(post["street_address"].gsub("\r\n", "<br/>"))
+      data << contact_line(post["locality"])
+      data << contact_line(post["postal_code"])
+      data << contact_line(post["world_location"])
+      return data if data.length.positive?
+      nil
```

Comment:
I would have thought this method would return nil automatically, without this? You could then probably remove the need for `return`